### PR TITLE
Fix #52: Update tor to 0.4.4.6

### LINKED ISSUE #52: Upgrade to Tor 0.4.4.6
See https://bravesoftware.slack.com/archives/C85NV6NSG/p1605641051064300?thread_ts=1604345872.052300&cid=C85NV6NSG for details.

### PR DESCRIPTION
**Pull request by jumde**
Fix #52

### CODE REVIEW

**Review by diracdeltas — DISMISSED** [review]
lgtm!

File: env.sh [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
 
 export BRAVE_TOR_VERSION="0"
-export TOR_VERSION="0.3.5.11"
+export TOR_VERSION="0.4.4.6"
 
 export ZLIB_VERSION=1.2.11
-export LIBEVENT_VERSION=2.1.9-beta
-export OPENSSL_VERSION=1.0.2q
+export LIBEVENT_VERSION=2.1.11-stable
+export OPENSSL_VERSION=1.0.2u
```
Comment (darkdh) [inline env.sh:10]: How do we decide which version of libevent or openssl to use?

**Comment by Pernat1y** [comment]
It is already in pull request here:
https://github.com/brave/tor_build_scripts/pull/51

**Comment by jumde** [comment]
> What about this pool request? #51

Hi @Pernat1y - Thanks for your PR, I added a comment to your PR. If you can change that, we can squash and merge it.

Since OpenSSL 1.0.2 is deprecated - https://www.openssl.org/blog/blog/2019/11/07/3.0-update/ - I'm looking at updating it to 1.1.1h. I was able to get the binary working on macOS, but I'm running into some linker issues on linux and windows. I should have a fix today.

**Review by darkdh — APPROVED** [review]

**Comment by jumde** [comment]
Was resolved by a PR by an external contributor: https://github.com/brave/tor_build_scripts/pull/51/files - Work to update OpenSSL will be tracked as a separate PR.

**Pull request closed without merging**